.github/workflows/pnnl_push_mirror.yml

```
@@ -6,17 +6,20 @@ jobs:
   to_gitlab:
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v1
-      - uses: spyoungtech/mirror-action@master
+      - uses: actions/checkout@v4
+        with:
+          # ref: ${{ github.head_ref }}
+          fetch-depth: 0
+      - uses: yesolutions/mirror-action@master
         with:
           REMOTE: ${{ secrets.GITLAB_REPO_URL }}
           GIT_USERNAME: ${{ secrets.GITLAB_USER }}
           GIT_PASSWORD: ${{ secrets.GITLAB_ACCESS_TOKEN }}
-          GIT_PUSH_ARGS: --push-option=ci.skip --tags --force --prune
+          #GIT_PUSH_ARGS: --push-option=ci.skip --tags --force --prune
```

hi @jaelynlitz, it isn't obvious to me why you are seeing an error related to invalid branch names. could you try adding the `--verbose` flag here? maybe that will shed some more light.

```suggestion
          GIT_PUSH_ARGS: --push-option=ci.skip --tags --force --prune --verbose
```

---

https://github.com/eagles-project/mam4xx/actions/runs/13018141683/job/36312271758?pr=406